### PR TITLE
feat: realdebrid library

### PR DESCRIPTION
**Pull request by Gaisberg**
- Try to match existing realdebrid torrents to a mediaitem
- Remove whitespaces
- Add logging to mdblist
- Api logging to be debug level

### CODE REVIEW

File: src/program/program.py [hunk 1]
```diff
@@ -71,12 +73,12 @@ def initialize_services(self):
...
             Updater: Updater(),
             Downloader: Downloader(),
-            # Depends on Symlinker having created the file structure so needs
-            # to run after it
-            SymlinkLibrary: SymlinkLibrary(),
+            SymlinkLibrary: SymlinkLibrary(), # Depends on Symlinker
             PostProcessing: PostProcessing(),
         }
 
+        self.services[RealDebridLibrary] = RealDebridLibrary(self.services[Downloader].services[RealDebridDownloader], self.services[TraktIndexer].run)
```
Comment (dreulavelle) [inline src/program/program.py:80]: With the Library, it needs the `match_existing_content` to be enabled for the app to run. Don't think that was intentional, as you wanted that to be optional right?

image

**Comment by spdustin** [comment]
Any plans to merge this? This would make migrating from pre-Riven libraries much easier.

**Comment by that-9uy** [comment]
+1

**Comment by Gaisberg** [comment]
> Any plans to merge this? This would make migrating from pre-Riven libraries much easier.

Be my guest, now that parsett is updated. I would want this to be automated below a certain ratio and store the rest to be mapped via frontend.

**Comment by davidemarcoli** [comment]
@Gaisberg I would also like this to be merged as it will help many users to transition to riven.
What needs to be done for it to be merged? New Table for the stuff that needs to be mapped in the frontend? How do we want it to show in the frontend? New Navigation option? Or somewhere in the summary tab under "unmapped items" or something?

It also keeps the door open for using your rd library as a "scraper".

**Comment by dreulavelle** [comment]
Personally I've been against this.. I kept it though as a proof of concept to test parsett/rtn against torrents to see how it would hold up.. but even with a threshold of 100% .. theres still room for error.

**Comment by Gaisberg** [comment]
Will be done seperately

**Pull request closed without merging**